std_msgs/Bool mode
---
std_msgs/Bool done
